### ISSUE TITLE
ParametricScalar Layer

### ISSUE THREAD

**Issue by sunweilunformatech** [keras-team/keras#16684]
It would be nice to add a ParametricScalar layer where learnable scalars are multiplied element-wise with the input similar to the PReLU activation function. Although this layer can be implemented as a custom layer, it is a commonly used module to merge multiple results together with learnable weights. A sample implementation is provided as follows:
```
class ParametricScalar(keras.layers.Layer):
    def __init__(self, alpha_initializer='ones', shared_axes=None, **kwargs):
        super().__init__(**kwargs)
        self.alpha_initializer = keras.initializers.get(alpha_initializer)
        if shared_axes is None:
            self.shared_axes = None
        elif not isinstance(shared_axes, (list, tuple)):
            self.shared_axes = [shared_axes]
        else:
            self.shared_axes = list(shared_axes)

    def get_config(self):
        return {'act_set': self.act_set}

    def build(self, input_shape):
        param_shape = list(input_shape[1:])
        if self.shared_axes is not None:
            for i in self.shared_axes:
                param_shape[i - 1] = 1
        self.alpha = self.add_weight(
            shape=param_shape,
            name='alpha',
            initializer=self.alpha_initializer)
        
    def get_config(self):
        config = {
            'alpha_initializer': keras.initializers.serialize(self.alpha_initializer),
            'shared_axes': self.shared_axes
        }
        base_config = super().get_config()
        return dict(list(base_config.items()) + list(config.items()))

    def call(self, inputs):
        return inputs * self.alpha```

**Comment by sunweilunformatech** [comment]
Thanks for considering it. Yeah, it works for me as long as the other issue on loading keras models with custom layers bug will be fixed in the next release which I reported here: [](https://github.com/keras-team/keras/issues/16685). But it is blocking me from upgrading me TF version (which is also fine for now, but I couldn't use any new features in TF2.6+) as some trained models with this layer fails to load.

For the usage of such layer, it is quite helpful in preserving the original signals in a lot of cases. For example, to lean f(x) = x > 0.5. Any normalization we put in f would destroy x's original strength and make the model unable to learn this simply task. But if we weight the normalized signal and the original signal using this scalar layer and sum them, it will have this ability. (Although a dense layer could do the same, it is a lot more expensive and prone to overfitting.) Maybe just my use case, but it is something to consider I think.
